sdk/go/message.go

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) 2018
+// Mainflux
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package sdk
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+// Default msgContentType is SenML
```

Comment:
This line is self explanatory. Remove the comment. Maybe also rename `msgContentType` in `defContentType`

Reply:
`msgContentType` is global var with is manipulated by a setter. It can not be called `defContentType` - it is not a constant, but a variable.

I find this comment useful, as nowhere else it is mentioned that SenML is default, and we can not see thie from other constants or a process of fetching value from env vars or similar.

Reply:
You are right for this def I didn't saw it... But the comment still obvious, Anyway, is just a comment it's fine.